README.md

```diff
@@ -1,5 +1,14 @@
 # ecg-gan
 
+Take a look at this [wandb run](https://wandb.ai/sjanas/ECG%20GAN/runs/ljdkb8p4?workspace=user-sneakyinsect) for some baseline results.
+
```

Comment:
Can you use a run where the results look more ecg-ish? I think this one is the best I've seen for long samples: https://wandb.ai/sjanas/ECG%20GAN/runs/bpr8skng 

Can you also make sure that it's straightforward to reproduce the same run? Mybe even set it to default (i.e. `python train.py` should produce it)

Reply:
done, `python train.py` reproduces the run